clickhouse-admin/api/src/lib.rs

```diff
@@ -136,4 +150,17 @@ pub trait ClickhouseAdminSingleApi {
     async fn init_db(
         rqctx: RequestContext<Self::Context>,
     ) -> Result<HttpResponseUpdatedNoContent, HttpError>;
+
+    /// Retrieve time series from the system database. The value the average of all
```

Comment:
The second sentence appears to be typo'd. Not sure what you are trying to say. 

Also, I'd move that sentence after a blank line, so that the doc comments have a short first line. Same for the method above.

Reply:
lol, idk what was happening there 😄